packages/frontend/editor-ui/src/stores/builder.store.ts

```diff
@@ -37,6 +38,7 @@ export const useBuilderStore = defineStore(STORES.BUILDER, () => {
 	const streaming = ref<boolean>(false);
 	const assistantThinkingMessage = ref<string | undefined>();
 	const streamingAbortController = ref<AbortController | null>(null);
+	const initialGeneration = ref<boolean>(false);
```

Comment:
IMO it would be cleaner for this to be a computed property instead of passing it around like we do now